### DISCUSSION TITLE
Bug report v8.0 beta - Bluetooth

### DISCUSSION THREAD

**Discussion by doobes** [elementary/triage#581] [Q&A]
I've got v8.0 beta installed on a Dell I spiron 7437 laptop. 

After the recent update, I can no longer access Bluetooth settings. Clicking on the icon in the upper right hand corner opens the Wi-Fi daemon.

Accessing via the settings page has the same effect. 

Consider this a bug report

**Comment by danirabbit** [comment]
Issue reports are tracked on individual projects using the "Issues" tab. Discussion posts cannot be tracked as issues.

Please see this [relevant Q&A post](https://github.com/orgs/elementary/discussions/621) and the [contributor guide](https://docs.elementary.io/contributor-guide/feedback/reporting-issues) for more info
